### ISSUE TITLE
trainset graphs

### ISSUE THREAD

**Issue by SPChytas** [cmavro/GNN-RAG#9]
Hello, and congratulations on your work. Is it possible to include the output of the first stage (the corresponding knowledge sub-graphs) for the train and val set too?

**Comment by cmavro** [comment]
Hi, thanks for the interest in our work!

Regarding training/validation sets, the ground-truth answers are known so we evaluate whether the GNN retrieves these answers. Thus, you can either use (i) the shortest paths between question and answer nodes, or (ii) the paths retrieved by GNN (you can do this during evaluation in this [line](https://github.com/cmavro/GNN-RAG/blob/28d31bc0db3d54f0930e6125f1c98b5b1ace1ba4/gnn/train_model.py#L204C1-L204C110) for `valid_data` and `train_data`, setting `write_info=True`).

**Comment by SPChytas** [comment]
Thanks. I am interested in option (ii), i.e., use the paths that the GNN has retrieved. I tried to follow the instructions: _"... or you can skip this folder and use directly the GNN output (retrieved answer nodes) that we computed (llm/results/gnn)."_ but I can't find the retrieved paths, beyond the test set.
